manifest/product-config.json

```diff
@@ -6,7 +6,7 @@
             "release_name": "Couchbase Sync Gateway",
             "production": true,
             "interval": 30,
-            "go_version": "1.13.4",
+            "go_version": "1.16.6",
```

Comment:
Couple more places to bump:
- `.github/workflows/ci.yml`
- `Jenkinsfile`

Reply:
Ah yeah hadn't considered the tests. Thanks